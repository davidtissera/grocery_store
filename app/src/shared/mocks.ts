// Define mocks here
const mocks = {};

export default mocks;
