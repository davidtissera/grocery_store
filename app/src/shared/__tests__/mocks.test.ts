import { products } from "../mocks";

describe("mocks tests", () => {
  describe("products mock", () => {
    it("should match snapshot", () => {
      expect(products).toMatchSnapshot();
    });
  });
});
