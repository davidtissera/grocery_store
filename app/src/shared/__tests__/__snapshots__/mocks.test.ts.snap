// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mocks tests products mock should match snapshot 1`] = `
Array [
  Object {
    "amount_of_products_with_discount": 2,
    "cost": 3.97,
    "discount_cost": 2.5,
    "name": "Milk",
  },
  Object {
    "amount_of_products_with_discount": 3,
    "cost": 2.17,
    "discount_cost": 2,
    "name": "Bread",
  },
  Object {
    "cost": 0.99,
    "name": "Banana",
  },
  Object {
    "cost": 0.89,
    "name": "Apple",
  },
]
`;
